conda_anaconda_tos/path.py

```diff
@@ -34,6 +34,42 @@
 #: Environment metadata directory. Located in the current conda environment.
 ENV_TOS_ROOT: Final = "$CONDA_PREFIX/conda-meta/tos"
 
+#: Locations safe from conda package updates (system-wide and user directories).
+SAFE_TOS_ROOTS: Final = tuple(
+    filter(
+        None,
+        (
+            SITE_TOS_ROOT,
+            "/var/lib/conda/tos" if not on_win else None,
+            "$XDG_CONFIG_HOME/conda/tos",
+            "~/.config/conda/tos",
+            USER_TOS_ROOT,
+            "$CONDATOS",
```

Comment:
Where do these hard-coded paths come from?

Since the backups aren't config files, `XDG_DATA_HOME` seems more appropriate (and correspondingly, `~/.local` instead of `~/.config`: https://specifications.freedesktop.org/basedir-spec/latest/#variables

For Windows, the data directories implemented in `platformdirs` are good places to add.